generators/app/files.js

```diff
@@ -21,9 +25,13 @@ module.exports.INDEX = 'index.js';
 module.exports.APP = `${module.exports.APP_PATH}/index.js`;
 module.exports.FASTLANE_EV = `${module.exports.IOS_PATH}/fastlane/config/.env`;
 
-// TESTING FILES
-module.exports.TESTS = '__tests__';
-module.exports.MOCKS = '__mocks__';
+// FONTS
+module.exports.MONTSERRAT_BOLD = `${module.exports.FONTS_PATH}/Montserrat-Bold.ttf`;
+module.exports.MONTSERRAT_BOLD_ITALIC = `${module.exports.FONTS_PATH}/Montserrat-BoldItalic.ttf`;
+module.exports.MONTSERRAT_ITALIC = `${module.exports.FONTS_PATH}/Montserrat-Italic.ttf`;
+module.exports.MONTSERRAT_SEMIBOLD = `${module.exports.FONTS_PATH}/Montserrat-SemiBold.ttf`;
+module.exports.MONTSERRAT_SEMIBOLD_ITALIC = `${module.exports.FONTS_PATH}/Montserrat-SemiBoldItalic.ttf`;
+module.exports.MONTSERRAT = `${module.exports.FONTS_PATH}/Montserrat.ttf`;
```

Comment:
I think that it isn't necessary to add all font files. It is just fine if you declare the folder path. All fonts will be imported as well